packages/block-editor/src/components/global-styles/utils.js

```diff
@@ -589,6 +589,11 @@ export function getResolvedRefValue( ruleValue, tree ) {
 		return ruleValue;
 	}
 
+	/*
+	 * This converts references to a path to the value at that path
+	 * where the value is an array with a "ref" key, pointing to a path.
```

Comment:
```suggestion
	 * Where the rule value is an object with a 'ref' property pointing
	 * to a path, this converts that path into the value at that path.
```

I know this comment was taken from the existing one in the PHP class but it reads poorly to me. Could we improve it? Maybe something along the lines of the suggestion above?

Reply:
I tweaked the wording in [936f216](https://github.com/WordPress/gutenberg/pull/64128/commits/936f2163e9cb64ddd8cde57f135e4d8f37140d65). Feel free to revert or improve as desired.